addons/web/static/src/public/colibri.js

```diff
@@ -24,20 +24,33 @@ export class Colibri {
         this.core = core;
         this.interaction = new I(el, core.env, this);
         this.interaction.setup();
+        this.bufferedClicks = [];
     }
 
     async start() {
+        const bufferHandler = (ev) => {
+            this.bufferedClicks.push({ ev, target: ev.target });
+        };
+
+        this.el.addEventListener("click", bufferHandler, true);
         await this.interaction.willStart();
+        this.el.removeEventListener("click", bufferHandler, true);
```

Comment:
Aren't we at risk of missing events between this remove and `processContent`'s `addListener` ?
I guess not given that nothing is async in the meantime - just making sure to think about it.